app/src/main/java/com/ataulm/whatsnext/di/AppComponent.kt

```diff
@@ -47,6 +37,8 @@ interface AppComponent {
         @BindsInstance
         fun application(application: Application): Builder
 
+        fun component(letterboxdComponent: LetterboxdComponent): Builder
```

Comment:
I think I have to provide this here because `AppComponent` depends on it && it's not possible for Dagger to create this component on account of it also using a builder(?)